src/Elsa.Server/appsettings.Development .json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
we removed this file from source control, so it shouldn't be there as 'new file' - hopefully after this it won't cause any more issues with the file disappearing